#Service to query number of joint waypoints
---
int32 numWaypoints
